zproject/default_settings.py

```diff
@@ -352,6 +352,7 @@
 # When security-relevant links in emails expire.
 CONFIRMATION_LINK_DEFAULT_VALIDITY_DAYS = 1
 INVITATION_LINK_VALIDITY_DAYS = 10
+INVITATION_LINK_VALIDITY_MINUTES = 10 * 24 * 60
```

Comment:
I moved this to `computed_settings.py`, to make it a function of `INVITATION_LINK_VALIDITY_DAYS`